control/control_block.go

```diff
@@ -15,12 +15,12 @@ type controlBlockType string
 const (
 	blockEndpoint                   controlBlockType = "endpoint"
 	blockFilter                     controlBlockType = "filter"
-	blockTrapezoidalVelocityProfile controlBlockType = "trapezoidalVelocityProfile"
+	BlockTrapezoidalVelocityProfile controlBlockType = "trapezoidalVelocityProfile"
```

Comment:
A comment that these are the way we're exposing command inputs into the control config. We're going to have to think a little about exposing functions so users don't access them as freely when they're coding their own control blocks. Can we use the controlBlockType  iota instead in your code?

Reply:
not sure how to create the new blockConfig without exposing this type but I'll keep working on this